.github/contributing.md

```diff
@@ -186,16 +145,24 @@ All lua code should be done using all the best practices of proper lua using the
 - Don't repeat yourself.. if you're using the same operations in many different places convert them into a function with flexible variables
 - For distance checking loops set longer waits if you're outside of a range
 - Job specific loops should only run for players with that job, don't waste cycles
-- When possible don't trust the client, esspecially with transactions
-- Balance security and optimizations
+- When possible don't trust the client, esspecially with sensitive code like handling money
+- Prioritize security over performance, we don't want any security vulnerabilities
+- Make sure everything is as optimized as you can make it
+- Make sure the code is friendly for other developers to expand on or edit and that the feature is also friendly for the end user
 - [Consider this Lua Performance guide](https://springrts.com/wiki/Lua_Performance)
-- Use local varriables everywhere possible
+- Use local variables where possible
 - Make use of config options where it makes sense making features optional or customizable
-- Instead of `table.insert(myTable, "Value")` use `myTable[#myTable + 1] = "Value"`
-- Instead of `table.insert(ages, "bob", 30)` use `ages["bob"] = 30`
-
+- Instead of `table.insert(myTable, "value")` use `myTable[#myTable + 1] = "value"`
+- Instead of `table.insert(ages, "bob", 30)` use `ages.bob = 30` or `ages["bob"] = 30`
```

Comment:
Opportunity to add additional lines here for things like 

- using backticks to load models from name
- preferring Wait() over Citizen.Wait()

or anything else that we consider to be "common" knowledge passed around on discord but isn't written down in these docs.